ansible/roles/phantom/tasks/install_phantom.yml

```diff
@@ -26,6 +26,6 @@
 
 # installing apps takes 15+ minutes longer, so later we will install just the apps we need
 - name: run the phantom install script without apps
-  shell: printf "{{phantom_community_username}}\n{{phantom_community_password}}\n" | /opt/phantom/bin/phantom_setup.sh install --no-prompt --without-apps --version=4.10.6.61906-1
+  shell: printf "{{phantom_community_username}}\n{{phantom_community_password}}\n" | /opt/phantom/bin/phantom_setup.sh install --no-prompt --without-apps --version=5.0.1.66250-1
```

Comment:
Would it be a huge PITA to get the version from `yum` here so that changing the `phantom_repo_url` alone is enough to switch versions?

Reply:
Short answer: Maybe.
Longer answer: at this point in execution, we haven't yet configured the repo where the actual product lives. That happens in the script, which is one of the sole contents of the repo provided.

Reply:
Definitely a PITA then. :-D